### DISCUSSION TITLE
I'm unable to push my code to Github.

### DISCUSSION THREAD

**Discussion by YashiOS** [community/community#50891] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, I'm getting this error while pushing code to Github.

This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.

**Comment by neutrinomuon** [comment]
Hello, 

Well, this 'error' suggests that your repository has exceeded its storage limit for Large File Storage (LFS) on GitHub. Public repositories on GitHub have storage limits and monthly transfer limits:

https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-git-large-file-storage

If you exceed these limits, GitHub may restrict your access to the repository or ask you to upgrade to a paid plan. It's important to keep track of your repository's usage and to optimize your code and files to reduce storage and transfer needs if necessary.

I have an option, however, it might not be optimal. If you don't want to purchase additional packs, you can try to remove large files from your repository's history to reduce its size. Or, if you are using Git LFS to store large files, you may also want to consider using other cloud storage outside GitHub (e.g., Amazon S3 or Google Cloud). This can help reduce your repository's size and minimize the risk of hitting storage limits in the future.

I hope this helps you :),

**Reply by neutrinomuon** [reply]
Hello, Have you managed to solve this problem?
